src/sentry/utils/committers.py

```diff
@@ -246,19 +246,23 @@ def get_event_file_committers(
     if not commits:
         raise Commit.DoesNotExist
 
-    frames = event_frames or ()
+    frames = event_frames or []
+    munged = munged_filename_and_frames(event_platform, frames, "munged_filename")
+    if munged:
+        frames = munged[1]
```

Comment:
Instead of mutating, we call the function to get any optionally munged frames and conditionally assign it to be used downstream.